### ISSUE TITLE
Helm 3: helm inspect values does not show comments

### ISSUE THREAD

**Issue by troytop** [helm/helm#6354]
Output of `helm version`:
version.BuildInfo{Version:"v3.0.0-beta.2", GitCommit:"26c7338408f8db593f93cd7c963ad56f67f662d4", GitTreeState:"clean", GoVersion:"go1.12.9"}

Output of `kubectl version`: N/A
Cloud Provider/Platform (AKS, GKE, Minikube etc.): N/A

The new `helm show values` (was `helm inspect`) outputs the parsed values.yaml of the chart with comments stripped out. This is a problem for helm charts that rely heavily on comments to document the exposed options (instead of repeating that information in the README). 

I think default behavior should be the same as `helm inspect` and dump the values.yaml file verbatim. There's definitely value in presenting the parsed/stripped values as it does now, but this should probably be exposed with a flag (e.g. `--stripped`, `--parsed`, or `--no-comments`).

**Comment by bacongobbler** [comment]
`helm show values` was actually `helm get values` in Helm 2. `helm get values` only displayed the rendered values from the release, whereas `helm inspect` dumped the values file from a chart.

`helm inspect` still appears to display comments from my testing:

```
><> ./bin/helm version
version.BuildInfo{Version:"v3.0+unreleased", GitCommit:"bc42dca470be8495315d0b03ea695fb833245e99", GitTreeState:"clean", GoVersion:"go1.13"}
><> ./bin/helm inspect stable/wordpress | tail -n 10
### To 3.0.0

Backwards compatibility is not guaranteed unless you modify the labels used on the chart's deployments.
Use the workaround below to upgrade from versions previous to `3.0.0`. The following example assumes that the release name is `wordpress`:

```console
$ kubectl patch deployment wordpress-wordpress --type=json -p='[{"op": "remove", "path": "/spec/selector/matchLabels/chart"}]'
$ kubectl delete statefulset wordpress-mariadb --cascade=false
```

Thanks for the report!

**Comment by cnweaver** [comment]
I believe that this is still a problem; specifically `helm inspect values` appears to lose comments in v3 when it did not in v2. The test above (`helm inspect stable/wordpress | tail -n 10`) successfully shows the readme (which is all that is visible when looking at only the last 10 lines with `tail`), but this is different from comments in the values.yaml. I suggest instead comparing `helm2 inspect values stable/wordpress | head` to `helm3 inspect values stable/wordpress | head` (assuming `helm2` and `helm3` refer to binaries of the appropriate versions):

	$ helm2 version
	Client: &version.Version{SemVer:"v2.12.1", GitCommit:"02a47c7249b1fc6d8fd3b94e6b4babf9d818144e", GitTreeState:"clean"}
	Server: &version.Version{SemVer:"v2.14.2", GitCommit:"a8b13cc5ab6a7dbef0a58f5061bcc7c0c61598e7", GitTreeState:"clean"}
	$ helm2 inspect values stable/wordpress | head
	## Global Docker image parameters
	## Please, note that this will override the image parameters, including dependencies, configured to use the global value
	## Current available global Docker image parameters: imageRegistry and imagePullSecrets
	##
	# global:
	#   imageRegistry: myRegistryName
	#   imagePullSecrets:
	#     - myRegistryKeySecretName
	#   storageClass: myStorageClass

	$ helm3 version
	version.BuildInfo{Version:"v3.0.0-beta.3", GitCommit:"5cb923eecbe80d1ad76399aee234717c11931d9a", GitTreeState:"clean", GoVersion:"go1.12.9"}
	$ helm3 inspect values stable/wordpress | head
	affinity: {}
	allowEmptyPassword: true
	allowOverrideNone: false
	customHTAccessCM: null
	externalDatabase:
	  database: bitnami_wordpress
	  host: localhost
	  password: ""
	  port: 3306
	  user: bn_wordpress

I agree with troytop that having both modes of output available can be useful, but for a project I'm working on (which wraps helm) I really need to be able to provide the full values file, with comments, to users.

**Comment by bacongobbler** [comment]
re-opening and updated the title.

**Comment by bacongobbler** [comment]
This is due to a regression where the `.RawValues` field was removed. I've identified the issue and will be working on a fix for 3.0.
